src/audio/host-zephyr.c

```diff
@@ -736,7 +736,7 @@ int host_common_params(struct host_data *hd, struct comp_dev *dev,
 	struct dma_sg_config *config = &hd->config;
 	struct dma_sg_elem *sg_elem;
 	struct dma_config *dma_cfg = &hd->z_config;
-	struct dma_block_config dma_block_cfg;
+	struct dma_block_config *dma_block_cfg;
```

Comment:
from the description this sounds like a bug-fix, and it's also unrelated to the native driver transition? This could be a separate bug-fix PR instead?

Reply:
I'd argue that it is related to the transition. You do need this for the native drivers to work (on NXP platforms at least).

Reply:
Its a good find and fix :)

Reply:
@LaurentiuM1234 this is really a bugfix and we want to have this in another PR so that we can merge it faster.

Reply:
Created https://github.com/thesofproject/sof/pull/8867